R/drive_file.R

```diff
@@ -7,68 +7,17 @@
 #' @export
 #'
 drive_file <- function(id = NULL, ...) {
-  request <- build_drive_file(id = id, ...)
+  request <- build_request(
+    endpoint = "drive.files.get",
+    token = drive_token(),
+    params = list(...,
```

Comment:
OK don't hold this PR up. I've tried to sell you on my request-building approach from googlesheets2. If you buy what I'm selling, you can take that logic from there as well (the checking for repetition). In Sheets, some repetition is actually allowed but AFAICT that's not true for Drive.